apps/forklift/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
     - "KAFKA_LISTENERS=INSIDE://:9094,OUTSIDE://:9092"
     - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
-    - "KAFKA_CREATE_TOPICS=event-stream:1:1,streaming-transformed:1:1,streaming-dead-letters:1:1,streaming-persisted:1:1"
+    - "KAFKA_CREATE_TOPICS=event-stream:1:1,streaming-transformed:1:1,dead-letters:1:1,streaming-persisted:1:1"
```

Comment:
@jdenen Make this same change in your e2e branch for the e2e `docker-compose.yml` file